src/components/reports/ReportForm.tsx

```diff
@@ -104,6 +113,9 @@ const ReportForm: FC<Props> = (props: Props) => {
   };
 
   return (
+    <>
+    <ModalWrapper isError={modal?.isError} modalMessage={modal?.message} onClear={() => setModal(null)} />
```

Comment:
I would have expected the indentation to change here.. strange

Reply:
Ah yes, my auto formatter has stopped working for some reason. I must have missed this one! I'll make sure to get prettier set up again!